src/integrationTest/java/uk/gov/hmcts/darts/testutils/IntegrationBaseWithWiremock.java

```diff
@@ -19,5 +19,8 @@ public class IntegrationBaseWithWiremock extends IntegrationBase {
     @BeforeEach
     void clearStubs() {
         dartsGateway.clearStubs();
+
+        // populate the jkws keys endpoint with a global public key
+        tokenStub.stubExternalJwksKeys(DartsTokenGenerator.getGlobalKey());
```

Comment:
`clearStubs()` suggests we're resetting all the stubs, but here we're setting a new stub. Consider renaming `clearStubs()` to something more appropriate, e.g. `setUp()`.

Reply:
Done